README.md

```diff
@@ -1,40 +1,148 @@
+<!-- nextcloud-talk-jitzsi-bot README.md -->
+<!-- version: 0.1.1 -->
+
 # Nextcloud Talk Jitsi Bot
 
-A chat bot for Nextcloud Talk that creates [Jitsi](https://jitsi.org) video chat meetings. It also includes a framework for building Nextcloud Talk chatbots.
+A chat bot for Nextcloud Talk that creates [Jitsi](https://jitsi.org) video chat meetings.
+It also includes a framework for building Nextcloud Talk chatbots.
 
 ## Overview
 
-Nextcloud Talk Jitsi Bot creates ad-hoc Jitsi meetings from Nextcloud Talk. Just add the bot to the conversation, type `#videocall` or `#videochat` and get a link to a Jitsi meeting!
+Nextcloud Talk Jitsi Bot creates ad-hoc Jitsi meetings from Nextcloud Talk sessions.
+
+The talk sessions needs to be aware of the bot. Just create a new `Jitsi-Bot user` inside your trusted `nextcloud` domain.
+Its name needs to reflact the `BOT_NEXTCLOUD_USERNAME` environment variable. The bot binary
+will resolve it and interact via the `Nextcloud API` to the `Talk` sessions.
+
+When you start a talk session, please add the `Jitsi-Bot user` as a participant of your session.
+Once done, you can type `#videocall` or `#videochat`. A newly created link to a Jitsi meeting will be advertised!
 
 ## Installation
 
-### Go Package
+Precompiled binaries are provided from the following locations.
 
-A Go package [is available](https://pkg.go.dev/mod/github.com/pojntfx/nextcloud-talk-jitsi-bot).
+### Go module
+
+A Go package is available from githubs module repro [pkg.go.dev](https://pkg.go.dev/mod/github.com/pojntfx/nextcloud-talk-jitsi-bot).
 
 ### Docker Image
 
-A Docker image is available on [Docker Hub](https://hub.docker.com/r/pojntfx/nextcloud-talk-jitsi-bot).
+A Docker image is available at [Docker Hub](https://hub.docker.com/r/pojntfx/nextcloud-talk-jitsi-bot).
+
+Pull the image
+
+	docker pull hub.docker.com/pojntfx/nextcloud-talk-jitsi-bot
+
+### OCI Image
+
+An Open Containers Initiative (OCI) image is available at
+[Quay.IO](https://quay.io/rzerres/nextcloud-talk-jitsi-bot).
+
+Pull the image
+
+	podman pull quay.io/rzerres/nextcloud-talk-jitsi-bot:latest
+
+<!--
+#### Howto uploading to Quay.io
+
+Get the container-id to be uploaded
+
+	imageid=$(buildah image | grep $botname | awk -F " " '{ print $3 }')
+
+Tag the container to an image
+
+	buildah commit $imageid quay.io/username/$botname:latest
+
+Now login to Quay.io
+
+	podman login quay.io
+	username: <username>
+	password: <password>
+
+Finally upload
+
+	buildah push quay.io/username/$botname:latest
+
+-->
+
+
+## Build
```

Comment:
While this is currently relevant to the end user (there are no pre-built binaries after all because there is no CI/CD for now) this should probably be moved to `CONTRIBUTING.md` as well

Reply:
> While this is currently relevant to the end user (there are no pre-built binaries after all because there is no CI/CD for now) this should probably be moved to `CONTRIBUTING.md` as well

Again, see mey comment at https://github.com/pojntfx/nextcloud-talk-bot-framework/pull/4#discussion_r406713042

Reply:
> On another note, see #5 for more info on the changes on the horizon!

That is a nice way to solve the issue. Probably using JavaScript is just to fast prototype? I definetly have to have a closer look at gRPC

Reply:
JS is "required" because it's the only way I know to use puppeteer (a headless chromium management tool); since the service is very minimal the detrimental perfomance should pale in comparison to creating a new Chromium tab with every request ;)